code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -209,6 +209,8 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 			//at this point, all requirements for the reaction are satisfied. we can now react()
 
 			. |= reaction.react(src, holder)
+			if (!reaction.bypass_nobstop)
+				continue
```

Comment:
this won't work, by the time the hypernob reacts it breaks the reaction loop, so this code would actually never be reached

Reply:
it worked tho but the thing is, it's no longer stopping other gases from reacting

Reply:
trying to figure this

Reply:
my PR will fix this properly it's fine

Reply:
what pr

Reply:
oh that one, yeah you probally gonna do better than mine